app/Models/License.php

```diff
@@ -678,4 +678,21 @@ public function scopeOrderCompany($query, $order)
         return $query->leftJoin('companies as companies', 'licenses.company_id', '=', 'companies.id')->select('licenses.*')
             ->orderBy('companies.name', $order);
     }
+    /**
+     * Determines if a license is available for checkout if there are enough free seats using API REST
+     *
+     * @author [D.Valin] [<david.valin@deivishome.com>]
+     * @since [v3.0]
+     * @return boolean
+     */
+    public function availableForCheckout()
```

Comment:
I believe we already have a method that does this. See `freeSeat()`

Reply:
(Also, please don't use capital letters for variable names.)

Reply:
Dear Snipe,
I used the remaincount as it returns the global value betwen available and checkedout, and to go faster i the remaicount > 0 then there are available, freeseats returns a list of information, and for the API in where you are just requesting to checkout a licenes, given the assgined_to_type (user, asset) and the value of the user or asset, and optionally note then i thought iwould be faster, if the funtion returns true then i check out to the seat/user id and type and Snipe associates it, if there is not available, return false then i would not allow to checkout, just having a true or false, not a list.